packages/@office-iss/react-native-win32/src-win/Libraries/Components/ScrollView/ScrollView.win32.js

```diff
@@ -1202,6 +1248,8 @@ class ScrollView extends React.Component<Props, State> {
           scrollToEnd: this.scrollToEnd,
           flashScrollIndicators: this.flashScrollIndicators,
           scrollResponderZoomTo: this.scrollResponderZoomTo,
+          // TODO: Replace unstable_subscribeToOnScroll once scrollView.addEventListener('scroll', (e: ScrollEvent) => {}, {passive: false});
```

Comment:
Is this from upstream or should it map to an issue on our side? If on us, can you link the issue?

Reply:
This is a TODO from upstream! https://github.com/facebook/react-native/blob/a05466c5b712c713b690b891a8f6c9f4f100d9a9/packages/react-native/Libraries/Components/ScrollView/ScrollView.js#L1251